src/frontend/validations.ts

```diff
@@ -4,7 +4,7 @@ const subdomainRegexp = /^\w(?:[\w-]{0,61}\w)?$/;
 const jiraDomains = ["atlassian.net", "jira.com"];
 
 export const validJiraDomains = (jiraSubdomain: string, jiraDomain: string): boolean =>
-	jiraSubdomain &&
+	!!jiraDomains && !!jiraSubdomain &&
```

Comment:
Wouldn't this condition be always true? `!!jiraDomains` hence it is a constant within this file? 

Maybe you wanted to do `!!jiraDomain` 

Also what does it do? Checks that it is not undefined?

Reply:
- [ ] jiraDomain or jiraDomains

Reply:
whoops, yeah, suppose to be `jiraDomain`.  And yes, `!!` changes it to boolean type and not just truthy.